lib/executor/ramping_vus_test.go

```diff
@@ -36,6 +37,34 @@ func TestRampingVUsConfigValidation(t *testing.T) {
 	errs = c.Validate()
 	require.NotEmpty(t, errs)
 	assert.Contains(t, errs[0].Error(), "greater than 0")
+
+	t.Run("validate large VU shifts should fail", func(t *testing.T) {
+		t.Parallel()
+
+		const MaxRampingVUShift = 100000000
+		const HalfMaxRampingVUShift = MaxRampingVUShift / 2
+		const HalfMaxInt = math.MaxInt64 / 2
+
+		c.StartVUs = null.IntFrom(MaxRampingVUShift)
+		errs = c.Validate()
+		require.NotEmpty(t, errs)
+		assert.Contains(t, errs[0].Error(), "number of VU shifts is too large")
+
+		c.StartVUs = null.IntFrom(0)
+		c.Stages = []Stage{
+			{Target: null.IntFrom(HalfMaxRampingVUShift + 1), Duration: types.NullDurationFrom(12 * time.Second)},
```

Comment:
I'm pretty sure I misunderstood or missed something, but I'm a bit confused as to why this specific test case should fail, could you please clarify? 🙇🏻